src/widget/whotcuebutton.cpp

```diff
@@ -28,6 +29,7 @@ void WHotcueButton::setup(const QDomNode& node, const SkinContext& context) {
     } else {
         SKIN_WARNING(node, context) << "Hotcue value invalid";
     }
+    m_cueColorDimThreshold = context.selectInt(node, QStringLiteral("DimBrightThreshold"));
```

Comment:
This should not keep the default if the value is not defined in skins.

Reply:
done.